amundsen_application/static/js/ducks/user/sagas.ts

```diff
@@ -29,3 +37,29 @@ export function* getUserWorker(action: GetUserRequest): SagaIterator {
 export function* getUserWatcher(): SagaIterator {
   yield takeEvery(GetUser.ACTION, getUserWorker);
 }
+
+export function* getUserOwnWorker(action: GetUserOwnRequest): SagaIterator {
+  try {
+    const userOwn = yield call(getUserOwn, action.payload.userId);
+    yield put({ type: GetUserOwn.SUCCESS, payload: userOwn.own });
```

Comment:
See comment in `ducks/user/types.ts` about the payload being an object that contains the data we wanted, and not just the data itself.